routing/car_router.cpp

```diff
@@ -578,6 +581,28 @@ bool CarRouter::DoesEdgeIndexExist(Index::MwmId const & mwmId)
   return true;
 }
 
+bool CarRouter::AllMwmsHasRoutingIndex() const
+{
+  // The new routing section with IndexGraph was added in december 2016.
+  // If some mwm does not have this section, old cross mwm routing is used.
+  int64_t constexpr kFirstVersionWithRoutingIndex = 161206;
+  vector<shared_ptr<MwmInfo>> infos;
+  m_index.GetMwmsInfo(infos);
+  for (auto const & info : infos)
+  {
+    if (info->GetType() != MwmInfo::COUNTRY)
+      continue;
+
+    if (info->GetCountryName() == "minsk-pass")
```

Comment:
As far as I know minsk-pass is regenerated during maps re-generation, as World and WorldCoasts. It's not a good idea to introduce exceptions here.